apps/ledger-live-mobile/e2e/specs/languageChange.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { device, expect } from "detox";
+import PortfolioPage from "../models/wallet/portfolioPage";
+import SettingsPage from "../models/settings/settingsPage";
+import GeneralSettingsPage from "../models/settings/generalSettingsPage";
+import { loadConfig } from "../bridge/server";
+import { delay } from "../helpers";
+
+let portfolioPage: PortfolioPage;
+let settingsPage: SettingsPage;
+let generalSettingsPage: GeneralSettingsPage;
+
+describe("Change Language", () => {
+  beforeAll(async () => {
+    await loadConfig("1AccountBTC1AccountETH", true);
+    portfolioPage = new PortfolioPage();
+    settingsPage = new SettingsPage();
+    generalSettingsPage = new GeneralSettingsPage();
+  });
+
+  it("should open on Portofolio page", async () => {
```

Comment:
I'd remove this step. It's tested elsewhere and not relevant to this language test

Reply:
ok

Reply:
I can't see it being used elsewhere though